cms/envs/common.py

```diff
@@ -175,6 +175,7 @@
 
     # for consistency in user-experience, keep the value of the following 3 settings
     # in sync with the ones in lms/envs/common.py
```

Comment:
Please delete this comment. Note that only 2 of the 3 original settings still exist.